js/deer-config.js

```diff
@@ -798,7 +799,7 @@ export default {
                             deerUtils.globalFeedbackBlip(wait, `Please wait for this Gloss information to load.`, false)
                             return
                         }
-                        const glossID = parentDataElem.getAttribute("data-id") ? parentDataElem.getAttribute("data-id").replace(/^https?:/, 'http:') : ""
+                        const glossID = parentDataElem.getAttribute("deer-id") ? parentDataElem.getAttribute("deer-id") : ""
```

Comment:
how odd these are both there. I think this is leftover from the CSS filter in Newberry